### ISSUE TITLE
Error message not shown

### ISSUE THREAD

**Issue by andreabioli** [vaadin/vaadin-combo-box#470]
### Description
The `vaadin-combo-box` doesn't show the error message when setting 'invalid' property directly.
I've used the component in this way:

```
<vaadin-combo-box id="cmbCountry" label="Country" items="{{countries}}" item-label-path="name" item-value-path="guid"
    value="{{clientCountryGuid}}" error-message="You have to select an existing Country" invalid="{{clientCountryGuidInvalid}}">
</vaadin-combo-box>
```

### Expected outcome
With the invalid property set to true, all the component should be red, and the (red) error message shown.

### Actual outcome
Initially, the country is not selected, the invalid property has value 'true', all the component is red, but the error mesage is not shown.
For what I could see in the Chrome developer tools, there is a paper-input-error that has in the style visibility: hidden, that could be the reason (but I don't know for sure).

### Browsers Affected
- [x ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE 11
- [ ] iOS Safari
- [ ] Android Chrome

**Comment by andreabioli** [comment]
The problem is even easier to reproduce just creating this instance in a page:

<vaadin-combo-box label="Something" error-message="You have to select an existing Country" invalid>
</vaadin-combo-box>

**Comment by web-padawan** [comment]
Fixed in 8431f42
Also, error message is now displayed correctly in latest 3.x release.
